3_Risk.md

```diff
@@ -1,37 +1,47 @@
 # Metric: Risk
 
+Goal: Identify the risk profile of a project. 
+
 ## Human Factors
-Activity Metric | Description
+
+Goal: Identify the risk associated with the people involved in a project. 
+
+Metric | Question
 --- | ---
-Contributor Importance | Percentage of commits by individual contributors from identified organizations over time.
-Qualified Committers | Contributions over time and what components they commit to over time.
-User Dependency | Number of users who are aware that they depend on the software over time.
-Paid Developers | Number of paid developers in community over time.
+Contributor Importance | What is the percentage of commits by individual contributors from identified organizations?
+Qualified Committers | What are their contributions and to what components do they commit to?

```

Comment:
In this one, I don't find the relationship between the name of the metric and the description. But maybe the problem is with the name: "qualified committers" seems to me more like "who are the most qualified committers"...

Reply:
Thanks @jgbarah, I don't see how "qualified" is reflected in the question either. I removed "qualified".